tests/js/keyboardInputTests.js

```diff
@@ -113,12 +114,28 @@ https://github.com/gpii/universal/LICENSE.txt
         jqUnit.assertEquals(msg, expected, keyboardInput.container.hasClass(className));
     };
 
+    gpii.tests.firstDiscovery.keyboardInput.checkKeypress = function (expected, actual) {
+        jqUnit.assertEquals("keypress character should be \"" + expected + "\"", expected, actual);
+    };
+
+    gpii.tests.firstDiscovery.keyboardInput.checkUserInput = function (keyboardInput, expected) {
+        // Check both the model value and the HTML input value
+        jqUnit.assertEquals("userInput should be \"" + expected + "\"",
+                            expected, keyboardInput.model.userInput);
+        jqUnit.assertEquals("HTML input value should be \"" + expected + "\"",
+                            expected, keyboardInput.container.val());
+    };
+
     gpii.tests.firstDiscovery.keyboardInput.setUpTooltipTest = function (keyboardInput) {
         // Set the focus and tooltip state to a known starting point
-        $("#gpiic-tests-other-input").focus();
+        gpii.tests.firstDiscovery.keyboardInput.focusOther();
         keyboardInput.tooltip.close();
     };
 
+    gpii.tests.firstDiscovery.keyboardInput.focusOther = function () {
+        $("#gpiic-tests-other-input").focus();
```

Comment:
Thanks for letting me know about fluid.focus(). I've updated the tests to use it instead of the jQuery focus().